.github/workflows/ci.yml

```diff
@@ -58,6 +58,61 @@ jobs:
       - run: npm run init
       - run: npm run test
 
+  # The versions should be kept up-to-date with the latest LTS Node releases.
+  # They next need to be rotated October 2021. See
+  # https://github.com/nodejs/Release.
+  sass_spec:
+    name: 'JS API Tests | Node ${{ matrix.node_version }} | ${{ matrix.os }}'
+    runs-on: ${{ matrix.os }}-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu, windows, macos]
```

Comment:
Does this need `dart_channel: [stable]`?

Reply:
Actually we should probably mirror the tests section and just set `{sdk: stable}`.